### ISSUE TITLE
Docker hanging on 'Downloading Linux chromedriver'

### ISSUE THREAD

**Issue by erichobson** [jjjchens235/bing-rewards#332]
**Before posting issue...**

1. Run using non-headless mode: `python BingRewards -nhl`
2. Copy relevant output from `errors.log`

**Your Issue - include the details from the above steps:**

Running in Docker with docker-compose, I am running into an issue where it is hanging on` Downloading Linux chromedriver`. Anyone have any idea why?

```
Command line options selected:
email=None, search_type=remaining, driver=<class 'src.driver.ChromeDriverFactory'>, headless=True, cookies=False, nosandbox=True, telegram=False, discord=False, google_sheets=False, google_trends_geo=US

You selected remaining
Downloading Linux chromedriver version: 109.0.5414.25
```

**Comment by heximing** [comment]
I have the same issue. I think the python program wants to download driver 109, but my computer Edge version  is 108.

**Comment by Killerherts** [comment]
the version of browser on your computer does not matter. To me it looks like your running the cmd without -nsb the run cmd should be ```python BingRewards.py -nsb``` without the nsb the script with hang at the Chrome download

**Comment by erichobson** [comment]
That was my mistake, running it with `-nsb` is also causing the issue.

**Comment by Killerherts** [comment]
Can you please post full logs?

**Comment by erichobson** [comment]
The logs are empty unfortunately.

**Comment by Killerherts** [comment]
> The logs are empty unfortunately.
Yeah I am going to still have to figure out what is going on with the underlying permission issue. Sorry I have not been at my desk since before Christmas. I will still look into this if no one figures it out before I'm back.

**Comment by erichobson** [comment]
> > The logs are empty unfortunately.
> 
> That means I am going to still have to figure out what is going on with the underlying permission issue. Sorry I have not been at my desk since before Christmas. I will still look into this if no one figures it out before I'm back.

No problem at all, appreciate you looking into it. Enjoy your holiday!
